taipy/gui_core/_context.py

```diff
@@ -671,7 +671,8 @@ def __edit_properties(self, entity: t.Union[Scenario, Sequence, DataNode], data:
 
     @staticmethod
     def sort_by_creation_date(entity: t.Union[Scenario, Cycle]):
```

Comment:
What is this function name?!? get_entity_date, no? + -> datetime.datetime

Reply:
ok

Reply:
but -> str

Reply:
get_entity_creation_date_iso